src/om_bootstrap/progress_bar.cljs

```diff
@@ -3,61 +3,65 @@
   work."
   (:require [om-bootstrap.types :as t]
             [om-tools.dom :as d :include-macros true]
-            [schema.core :as s])
-  (:require-macros [schema.macros :as sm]))
+            [schema.core :as s :include-macros true]
+            [om-bootstrap.util :as u]))
 
 ;; ## Schema
 
 (def ProgressBar
   (t/bootstrap
-   {:min s/Int
-    :now s/Int
-    :max s/Int
-    :label t/Renderable
-    :sr-only? (s/named s/Bool "Screenreader-only? Hide the label?")
-    :striped? s/Bool
-    :active? s/Bool}))
+   {:now s/Int
+    (s/optional-key :min) s/Int
+    (s/optional-key :max) s/Int
+    (s/optional-key :label) t/Renderable
+    (s/optional-key :sr-only?) (s/named s/Bool "Screenreader-only? Hide the label?")
+    (s/optional-key :striped?) s/Bool
+    (s/optional-key :active?) s/Bool
+    (s/optional-key :nested?) (s/named s/Bool "Specify this for a nested ProgressBar inside a stacked ProgerssBar.")}))
```

Comment:
I added this key - false by default - to handle the case of stacked progress bars. The react-bootstrap guys use keys (https://github.com/react-bootstrap/react-bootstrap/blob/master/docs/examples/ProgressBarStacked.js), but I thought the :nested? boolean might be more appropriate for us